c/validation/adbc_validation_statement.cc

```diff
@@ -263,6 +267,11 @@ void StatementTest::TestSqlIngestLargeString() {
       NANOARROW_TYPE_LARGE_STRING, {std::nullopt, "", "", "1234", "例"}, false));
 }
 
+void StatementTest::TestSqlIngestStringView() {
+  ASSERT_NO_FATAL_FAILURE(TestSqlIngestType<std::string>(
+      NANOARROW_TYPE_STRING_VIEW, {std::nullopt, "", "", "1234", "例"}, false));
```

Comment:
Not sure how much this affects the test suite, but probably good to add a string here that will exceed the inline size (maybe even as a separate test)